server/server.js

```diff
@@ -28,40 +28,57 @@ app.get('/', (req, res) => {
  * 
  * 1. Verify account
  * 2. Get account
- * 3. TODO: Create session
- * 4. TODO: Set cookie
+ * 3. Create session
+ * 4. Set cookie
  */
-// app.post('/login', accountController.verifyUser, sessionController.setCookie, sessionController.startSession);
-app.post('/login', accountController.verifyAccount, accountController.getAccount, (req, res, next) => {
-    res.send(res.locals.account);
-});
+app.post('/login', sessionController.verifySession, 
+    accountController.verifyAccount, accountController.getAccount,
+    sessionController.startSession, sessionController.setCookie,
```

Comment:
setCookie => startSession

Reply:
Actually the start session creates the token